POS/Operation/02_SelectOffer.md

```diff
@@ -87,34 +155,116 @@ Offers for POS are created, see [Manage offers for POS](../Integration/07_Manage
     > [Info] The search already provides you with proposals for your number as you type.
 
 
-3. Select the desired offer by clicking the corresponding product in the list.   
-    The offer is added to the bill list on the right side.
+3. Select the desired offer by clicking the corresponding offer in the list.   
+    The offer is added to the bill list if no further features such as an [age verification](#verify-the-age) or a [product option](#add-a-product-option) are interposed. 
 
-    > [Info] The number remains in the search bar when you add a product to the bill list. When you click the search bar with that number again, the search results list for this number is displayed again.
+    > [Info] The number remains in the search bar when you add an offer to the bill list. When you click the search bar with that number again, the search results list for this number is displayed again.
 
 
+### Select a pre-printed voucher offer 
 
-## Use single scan
+Select a voucher offer that allows you to identify and charge a pre-printed voucher. You can then scan the voucher code with a barcode scanner or add the card code manually. 
```

Comment:
Select a voucher offer that allows you to identify and charge a pre-printed voucher. -> Damit verstehe ich nicht ganz, was ein pre-printed voucher ist.